src/python/system/new_process.py

```diff
@@ -361,7 +361,19 @@ def run_and_wait(self,
     return result
 
 
-class UnicodeProcessRunner(ProcessRunner):
+class UnicodeProcessRunnerMixin(object):
+  """Mixin for process runner subclasses to output unicode output."""
+
+  def run_and_wait(self, *args, **kwargs):  # pylint: disable=arguments-differ
+    """Overridden run_and_wait which always decodes the output."""
+    result = ProcessRunner.run_and_wait(self, *args, **kwargs)
+    if result.output is not None:
+      result.output = utils.decode_to_unicode(result.output)
+
+    return result
+
+
+class UnicodeProcessRunner(UnicodeProcessRunnerMixin, ProcessRunner):
   """ProcessRunner which always returns unicode output."""
 
   def run_and_wait(self, *args, **kwargs):  # pylint: disable=arguments-differ
```

Comment:
Why did we create UnicodeProcessRunnerMixin ? it has the same run_and_wait as UnicodeProcessRunner ?

Reply:
Oops, forgot to delete the run_and_wait in the Runner.

This is needed for subclasses of ProcessRunner (e.g. for minijail). We cannot just do e.g.

```python
class MinijailLibFuzzerRunner(UnicodeProcessRunner, MinijailProcessRunner):
```

As `UnicodeProcessRunner` may override things that we need from `MinijailProcessRunner`